### PR TITLE
Disable drafts tutorial when running e2e tests

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Trying to fix some more failures that are coming our way.

### CODE REVIEW

**Review by cpoile — APPROVED** [review]
Thank you -- did you get notified ahead of time? That's awesome if you did.

**Comment by streamer45** [comment]
> Thank you -- did you get notified ahead of time? That's awesome if you did.

Unfortunately not. I was testing some changes to the pipeline and suddenly got 30 tests failed out of nowhere :grimacing:

**Comment by cpoile** [comment]
Someday, someday. :)

File: e2e/global-setup.ts [hunk 1]
```diff
@@ -121,6 +121,7 @@ async function globalSetup(config: FullConfig) {
             data: [
                 {user_id: user.id, category: 'recommended_next_steps', name: 'hide', value: 'true'},
                 {user_id: user.id, category: 'insights', name: 'insights_tutorial_state', value: '{"insights_modal_viewed":true}'},
+                {user_id: user.id, category: 'drafts', name: 'drafts_tour_tip_showed', value: 'true'},
```
Comment (streamer45) [inline e2e/global-setup.ts:124]:
```suggestion
                {user_id: user.id, category: 'drafts', name: 'drafts_tour_tip_showed', value: '{"drafts_tour_tip_showed":true}'},
```

**Pull request merged**